GOVERNANCE.md

```diff
@@ -0,0 +1,96 @@
+  English proficiency levels, etc.
+
+- **All communications are public**.
+  There are no permanent private channels where maintainers discuss "internal" matters.
+  Occasional private chat or email messages may be exchanged,
+  e.g. when setting up services that require passwords,
+  but otherwise all communications that impact the project
+  will either happen in issue and PR discussions,
+  or in the [Gitter chat room](https://gitter.im/tldr-pages/tldr)
+  (which is open to all, and publicly logged).
+
+- **All decisions are made by community consensus.**
+  This does not mean there has to be unanimity,
+  nor that decisions result from vote counts.
+  What it means is that every interested member of the community can voice their thoughts,
+  and different positions are ideally resolved via
+  [informed consent](https://en.wikipedia.org/wiki/Sociocracy#Consent_vs._consensus)
+  of the involved people, who accept the collective decision
+  as "good enough for now, safe enough to try".
+
+
+## II. Role transitions
+
+The main goal of these principles is to support a continuous replenishing of the management team
+via a **smooth transition flow between community roles** —
+from newcomer, to occasional contributor, to regular contributor, to maintainer.
+This way the project can adapt in a flexible way
+to the the natural variations in availability and interest of its contributors,
+ensuring long-term resilience,
+and avoiding [single points of failure](https://en.wikipedia.org/wiki/Bus_factor).
+
+To this end, rather than assigning roles and tasks to people,
+these guidelines instead aim to **recognize the work that people already do**.
+Everyone is therefore encouraged to get involved and contribute to the project in whatever way they prefer,
+and we will strive to **get barriers out of the way** of these contributions.
+
+To ensure that these processes are transparent, predictable, and free from subective judgment,
+the metrics used are simple, objective, and publicly available.
+
+- Regular contributors shall be recognized as collaborators in the organization.
+
+    - Specifically: once a contributor has had **5 pull requests merged**,
+      they should be invited to become a
+      [**member of the tldr-pages organization**](https://github.com/orgs/tldr-pages/people).
+      This means they will be able to push commits to all of the organization's repositories,
+      merge PRs, label and close issues, among other things.
+      Note: All members of the tldr-pages organization must make their membership public.
+
+- Members of the organization who demonstrate interest in performing maintainership tasks,
+  by reviewing and/or merging PRs, responding to and labeling issues, and generally doing project maintenance work,
+  shall be made part of the maintenance team, and their name added to the list of current maintainers
+  in the [MAINTAINERS.md](MAINTAINERS.md) file.
```

Comment:
I recommend to recognize non code contributions as well. We've had people being active on our code of conduct, performing usability tests or just "joking around making the community more happy".

Reply:
@sils fully agreed. I'm not sure where you're getting the "code-only contributions" vibe from, though. Can you quote the specific passages that you'd suggest rephrasing? Or was this meant as a general comment to the whole document?

Reply:
you seem to be measuring activity by PRs made or reviewed. I think we have an additional clause that is vague by design and allows the maintainers to rank up anyone for arbitrary awesome stuff done.

Reply:
Oh, I see what you mean. While this text explicitly mentions a variety of maintenance-related activities:

> reviewing and/or merging PRs, responding to and labeling issues, and generally doing project maintenance work

...the actual metrics only take PRs merged and reviewed into account. The first reason for this is that those are clear, transparent metrics that are easy to check; the second, and most important reason IMO, is to avoid giving these role transitions a meaning of recognition of past work, i.e. make them into some sort of promotion/demotion. Instead, they should occur in a casual, natural manner<sup>†</sup>, in either direction, without connotations of evaluation or judgment (positive or negative). For example, nobody has to second-guess why person X was added and person Y wasn't, nor why person Z was removed. I think you'll agree that it's very hard to measure "awesomeness" objectively :) and in a way that everyone will agree.

**That said**, such a vague clause is already possible, via the "all decisions are made by community consensus" item. As a concrete example, I was planning on starting a discussion, after this PR, to propose making @rprieto, as the project's creator, not affected by the inactivity guidelines.

That way we allow this document to remain a set of loose guidelines, rather than an exhaustive list of procedures. Adding exceptions for all edge cases would make it increasingly more verbose and, ironically, less flexible.

<sup> †  In fact, ideally these role transitions would happen automatically, as is the case for example with StackOverflow, where privileges are distributed automatically based on karma, and even the moderators are elected by the entire community.</sup>